src/UniversalApp.js

```diff
@@ -56,7 +57,12 @@ const initServer = function(app, routes, apolloServerOptions, production) {
                 : context
           })
             .then(html => {
-              res.status(responseStatusCode);
+              if (
+                res.statusCode === DEFAULT_STATUS_CODE &&
```

Comment:
I think we can also use this constant to replace the magic number - 200 in L41

Reply:
I have added a new commit to use `DEFAULT_STATUS_CODE` to replace 200 in c8b867d. Please review again.